testutils/memory.go

```diff
@@ -0,0 +1,120 @@
+package testutils
+
+import (
+	"github.com/src-d/metadata-retrieval/github/graphql"
+	"gopkg.in/src-d/go-log.v1"
```

Comment:
we can also split these ones  because `"github.com/src-d/metadata-retrieval/github/graphql"` belongs to this repo, but you can skip it as @smacker mentioned